recipes/aruco/3.x.x/test_package/conanfile.py

```diff
@@ -0,0 +1,25 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [False, True], "fPIC": [False, True]}
+    default_options = {"shared": False, "fPIC": True}
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def build(self):
```

Comment:
As @prince-chrismc  mentioned. `shared` and `fPIC` matter most when building libraries, not executables.
OTOH, I think `fPIC` is also useful for executables: `fPIE` for enaing ASLR.
```suggestion
    settings = "os", "compiler", "arch", "build_type"

    def build(self):
```